lib/rules/no-unused-vars.js

```diff
@@ -858,10 +861,341 @@ module.exports = {
+                                : getDefinedMessageData(unusedVar),
+                            suggest: [
+                                {
+                                    messageId: "removeVar",
+                                    data: {
+                                        varName: unusedVar.name
+                                    },
+                                    fix(fixer) {
+                                        const id = unusedVar.identifiers[0];
+                                        const parent = id.parent;
+                                        const tokenBefore = sourceCode.getTokenBefore(id);
+                                        const tokenAfter = sourceCode.getTokenAfter(id);
+
+                                        /**
+                                         * Gets fixes in variable declarations and function parameters
+                                         * @param {ASTNode} node parent node of identifier
+                                         * @returns {Object} fixer object
+                                         */
+                                        function fixVariables(node) {
+                                            if (node.parent.type === "VariableDeclarator") {
+                                                if (node.parent.parent.declarations.length === 1) {
+                                                    return fixer.removeRange(node.parent.parent.range);
+                                                }
+
+                                                if (sourceCode.getTokenBefore(node.parent).value === ",") {
+                                                    return fixer.removeRange([sourceCode.getTokenBefore(node.parent).range[0], node.parent.range[1]]);
+                                                }
+
+                                                return fixer.removeRange([node.parent.range[0], sourceCode.getTokenAfter(node.parent).range[1]]);
+                                            }
+
+                                            if (node.parent.type === "FunctionDeclaration" || node.parent.type === "FunctionExpression") {
+                                                if (node.parent.params.length === 1) {
+                                                    return fixer.removeRange(node.range);
+                                                }
+
+                                                if (sourceCode.getTokenBefore(node).value === "(" && sourceCode.getTokenAfter(node).value === ",") {
+                                                    return fixer.removeRange([node.range[0], sourceCode.getTokenAfter(node).range[1]]);
+                                                }
+
+                                                return fixer.removeRange([sourceCode.getTokenBefore(node).range[0], node.range[1]]);
+                                            }
+
+                                            return;
```

Comment:
```suggestion
                                            return null;
```

To fix the lint errors.

Reply:
Done!